api/internal/crawl/index/elasticsearch.go

```diff
@@ -219,7 +222,8 @@ func (idx *index) Put(uniqueID string, doc interface{}) error {
 			Body:       bytes.NewReader(body),
 			DocumentID: uniqueID,
 		}
-		res, err := req.Do(idx.ctx, idx.client)
+		var res *esapi.Response
+		res, err = req.Do(idx.ctx, idx.client)
```

Comment:
this isn't actually fixing anything, i's just removing a shadow
suggest adding an if err != nil here.

Reply:
ack!